build.sbt

```diff
@@ -128,8 +128,8 @@ lazy val testing = crossProject(JSPlatform, JVMPlatform)
   .settings(
     name := "log4cats-testing",
     libraryDependencies ++= Seq(
-      "org.typelevel" %%% "cats-effect"     % catsEffectV,
-      "ch.qos.logback"  % "logback-classic" % logbackClassicV % Test
+      "org.typelevel" %%% "cats-effect" % catsEffectV,
+      "ch.qos.logback"                  % "logback-classic" % logbackClassicV % Test
```

Comment:
Jeez. This looks bizarre a bit.

Reply:
I don't love that it's vertically aligning those.